src/Features/CSharp/Portable/Organizing/Organizers/RecordDeclarationOrganizer.cs

```diff
@@ -0,0 +1,42 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Composition;
+using System.Threading;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.Host.Mef;
+using Microsoft.CodeAnalysis.Organizing.Organizers;
+
+namespace Microsoft.CodeAnalysis.CSharp.Organizing.Organizers
+{
+    [ExportSyntaxNodeOrganizer(LanguageNames.CSharp), Shared]
+    internal class RecordDeclarationOrganizer : AbstractSyntaxNodeOrganizer<RecordDeclarationSyntax>
+    {
+        [ImportingConstructor]
+        [Obsolete(MefConstruction.ImportingConstructorMessage, error: true)]
+        public RecordDeclarationOrganizer()
+        {
+        }
+
+        protected override RecordDeclarationSyntax Organize(
+            RecordDeclarationSyntax syntax,
+            CancellationToken cancellationToken)
+        {
+            return syntax.Update(
+                syntax.AttributeLists,
+                ModifiersOrganizer.Organize(syntax.Modifiers),
+                syntax.Keyword,
+                syntax.Identifier,
+                syntax.TypeParameterList,
+                syntax.ParameterList,
+                syntax.BaseList,
+                syntax.ConstraintClauses,
+                syntax.OpenBraceToken,
+                MemberDeclarationsOrganizer.Organize(syntax.Members, cancellationToken),
+                syntax.CloseBraceToken,
+                syntax.SemicolonToken);
+        }
```

Comment:
can you jsut do `syntax.WithModifiers(...).WithMembers(...);`?

Reply:
@CyrusNajmabadi Sounds reasonable. But I followed the same pattern used in other implementations in the same folder to be consistent with them. If I'd change it here, I'd change it everywhere in the same folder too. Do you want me to update all of them in this PR? or keep it for later PR?

Reply:
Two allocations is fine here :-)

Reply:
> I also would like to know where this feature exist in Visual Studio.

The feature never shipped. But it has come up recently as something we might like to resurrect